src/main/java/ratpack/zipkin/internal/RatpackCurrentTraceContext.java

```diff
@@ -23,7 +23,12 @@ public TraceContext get() {
     return registrySupplier.get()
         .maybeGet(TraceContextHolder.class)
         .map(TraceContextHolder::getContext)
-        .orElse(null);
+        .orElseGet(() -> (Execution.isManagedThread())
+          ? Execution.current()
```

Comment:
I checked and I agree that was redundant, we only needed to make the `TraceContextHolder` / `TypedValue` class to be public. my bad